### ISSUE TITLE
Link-local neighbor address fails validation

### ISSUE THREAD

**Issue by samip5** [natesales/pathvector#69]
**Describe the bug**
Unable to have neighbors with link-local addresses.

**Environment**
1. Output from `pathvector version`:
```
INFO[0000] Pathvector 5.5.2 commit 4c44c80ed53032dbf9a8b16a9bcc57ae16c0fe88 date 2021-09-12T08:55:22Z
```
2. OS version: `Debian Bullseye (Debian 11)`
3. Minimum reproducible example of your Pathvector config file formatted with [triple backticks for formatting](https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks)
```yaml
asn: 213021
router-id: 192.168.10.1
peers:
  home_router:
    asn: 213021
    next-hop-self: true
    allow-local-as: true
    mp-unicast-46: true
    neighbors:
      - fe80:3::2%wg1
```

**To Reproduce**
1. Copy this config to pathvector.yml...
2. Run `pathvector -d generate`
3. See error

```
bird: ./AS213021_HOME_ROUTER.conf:21:43 syntax error, unexpected '%'
FATA[0009] BIRD config validation: exit status 1 
```

**Expected behavior**
It wouldn't fail validation.

**Comment by aredan** [comment]
I think that peer on IPv6 with link-local is not supported in pathvector.
 Maybe you can assign a Global Addr to that interface and use that?  that way you don't need to specify an interface.

**Comment by samip5** [comment]
>  I think that peer on IPv6 with link-local is not supported in pathvector.
>  Maybe you can assign a Global Addr to that interface and use that?  that way you don't need to specify an interface.

It is now, the only thing missing was a way to tell it that its directly connected peer.

**Comment by samip5** [comment]
> As of fbc927e, Pathvector does support IPv6 link local peering! @samip5 Does that address your issue?

My issue was fixed by 4f537dada31e2c8c723761ba4aec607c75da9386.
